swift-generator/src/main/resources/templates/java/common.st

```diff
@@ -399,3 +414,7 @@ public void <field.javaSetterName>(final <field.javaType> <field.javaName>) {
     this.name = "<field.name>";
 }
 >>
+
+generatedComment(context) ::= <<
+// Generated by swift-generator from <context.sourceIDLPath> on <context.currentTime>
```

Comment:
swift-maven-plugin has configuration parameter idlFiles:directory which defines location of idl files. We can use this directory as a base and put idl file names relative to this directory. This is what antlr does for example.